django/contrib/postgres/forms/jsonb.py

```diff
@@ -52,3 +52,12 @@ def prepare_value(self, value):
         if isinstance(value, InvalidJSONInput):
             return value
         return json.dumps(value)
+
+    def has_changed(self, initial, data):
+        """
+        Return True if data differs from initial.
+        """
+        if super(JSONField, self).has_changed(initial, data):
```

Comment:
How about:
```
return super(JSONField, self).has_changed(initial, data) or str(initial) != str(data)
```
I think there's no need to repeat the docstring from the superclass but explaining why the `str()` comparison is needed would be useful.